homeassistant/helpers/llm.py

```diff
@@ -443,6 +450,9 @@ def _async_get_tools(
             for intent_handler in intent_handlers
         ]
 
+        if exposed_domains and WEATHER_DOMAIN in exposed_domains:
+            tools.append(WeatherForecastTool())
```

Comment:
Make sure that Weather is no longer included in `_get_exposed_entities`

Reply:
The weather entities report the current weather, and the tool reports the forecast, I think we need both

Reply:
We could change that though, include "current weather" in the tool as well. That way we don't always include all that info.